themes/helm/layouts/index.html

```diff
@@ -152,6 +153,12 @@ <h2 class="has-text-centered">{{ T "community_title" }}</h2>
     <div class="columns is-desktop community-boxes">
 
       <div class="column is-one-third-desktop">
+        <section class="box">
+          <h1 class="title">{{ T "community_next_release" }}</h1>
+          <dt>{{ T "community_next_release_version" | markdownify }} {{ $nextversion.version }}</dt>
+          <dt>{{ T "community_next_release_date" | markdownify }} {{ $nextversion.date }}</dt>
```

Comment:
I think the HTML tags should be different. I noticed there was not enough space below the dt and the bottom of the section in the visual display which lead me to this.

I think they needs to be a [`<dl>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl) that wraps `dt` aand `dd` tags. Or, something else.

In the following image see the difference in gap at the bottom of the box...
<img width="866" alt="Screen Shot 2020-09-28 at 10 34 01 AM" src="https://user-images.githubusercontent.com/62991/94446118-357bb300-0176-11eb-972b-495fc4a7f890.png">

Reply:
You are correct, I was missing the `<dl>` tags.
Instead of a two element in the list, I opted for a single element with a break.  That put less space between the version and the date lines.

What do you think?